### DISCUSSION TITLE
autosave

### DISCUSSION THREAD

**Discussion by Sarilein** [xournalpp/xournalpp#4539] [Q&A] [answered]
Hi there,
I have the following problem: I worked the last few days on a lecture summary but for some reason I never saved this file. Not even once. 
And of course my notebook restarted now, all files were closed and of course this particular Journal file isn't there anymore...So my question: Are there anywhere autosave files? But I'll guess it would have been necessary to save this file at least once, so that it has a name in repository, isn't it? Because when I look in the last openend foldert, it isn't listed there (of course...)

Hope you can enlighten me...Thanks.

**Comment by rolandlo** [comment, marked as answer]
When you haven't saved the file, autosave files are created in the Cache folder. Find its location on the [website](https://xournalpp.github.io/guide/file-locations/) depending on your OS.

**Reply by Sarilein** [reply]
Thank you!!

**Reply by Sarilein** [reply]
Okay....another question, I'm sorry:
 on my Surface the solution with given path is working...xournal created a cache folder in my users folder... But on my other Laptop, there isn't such a folder, although they have the same OS despite Windows 11 on my Surface and Windows 10 on my Laptop... Any Idea where the folder could be shifted there? 
With the explorer search I found some Appdata paths and folders but no such folder with NetCache.
